activerecord/test/models/tag.rb

```diff
@@ -5,3 +5,9 @@ class Tag < ActiveRecord::Base
 
   has_many :tagged_posts, :through => :taggings, :source => 'taggable', :source_type => 'Post'
 end
+
+class OrderedTag < Tag
+  self.table_name = "tags"
+
+  has_many :taggings, -> { order('taggings.id DESC') }, :foreign_key => 'tag_id'
```

Comment:
Here should have Ruby 1.9 syntax as well:

`has_many :taggings, -> { order('taggings.id DESC') }, foreign_key: 'tag_id'`